src/st/e_harpy.h

```diff
@@ -0,0 +1,347 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+extern EInit g_EInitHarpy;
+extern AnimationFrame D_us_80181F04[];
+extern AnimationFrame D_us_80181F10[];
+extern AnimationFrame D_us_80181F1C[];
+extern AnimationFrame D_us_80181F28[];
+extern AnimationFrame D_us_80181F3C[];
+extern AnimationFrame D_us_80181F48[];
+extern AnimationFrame D_us_80181F5C[];
+extern AnimationFrame D_us_80181F68[];
+extern s16 D_us_80181FA8[];
+
+void EntityHarpy(Entity* self) {
+    Entity* entity;
+    Entity* player;
+    s32 i;
+    s16 angle;
+    s32 posX;
+    s32 posY;
+    s32 velocity;
+
+    if ((self->step & 1) && (self->hitFlags & 3)) {
+        SetStep(0xC);
+    }
+
+    if (self->flags & 0x100 && self->step != 14) {
+        PlaySfxPositional(SFX_HARPY_DEATH);
+        SetStep(14);
+    }
+
+    switch (self->step) {
+    case 0:
+        InitializeEntity(g_EInitHarpy);
+        self->zPriority = 0x9C;
+        self->ext.harpy.timer = 0x80;
+        entity = self + 1;
+        CreateEntityFromCurrentEntity(0x39, entity);
```

Comment:
Can these IDs for create entity be added to the overlay EntityIds enum, even if we just make the E_UNK_39 or whatever for now (there's a few so I won't comment on each)